sdk/zksend/src/links/claim.ts

```diff
@@ -132,9 +132,8 @@ export class ZkSendLink {
 				...options,
 				keypair,
 				network:
-					options.network ?? parsed.searchParams.get('network') === 'testnet'
-						? 'testnet'
-						: 'mainnet',
+					options.network ??
```

Comment:
this needs changed on line 150 below too

Reply:
also I'd re-write this as the following so we don't make similar mistakes in the future :)

```
		const networkFromParams = parsed.searchParams.get('network') === 'testnet' ? 'testnet' : 'mainnet';
		const network = options.network ?? networkFromParams;
```

Reply:
@williamrobertson13 should be ready now